package.json

```diff
@@ -6,8 +6,13 @@
   "scripts": {
     "postinstall": "npm run wd:update",
     "wd:update": "webdriver-manager update",
-    "test:cucumber": "protractor ./cucumber/protractor.config.js",
-    "test:mocha": "protractor ./mocha/protractor.config.js"
+    "test:cucumber-firefox": "protractor ./cucumber/protractor-firefox.config.js",
+    "test:cucumber-chrome": "protractor ./cucumber/protractor-chrome.config.js",
+    "test:cucumber-parallel": "protractor ./cucumber/protractor-parallel.config.js",
+    "test:cucumber-grid": "protractor ./cucumber/selenium-grid.config.js",
+    "test:mocha": "protractor ./mocha/protractor.config.js",
```

Comment:
Could you please add possibility to execute mocha tests parallel way? And please add screenshot functionality to mocha tests too.

Reply:
done